keyboards/momoka_ergo/config.h

```diff
@@ -17,36 +17,28 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #pragma once
 
-#include "config_common.h"
-
 /* key matrix size */
 #define MATRIX_ROWS 14
 #define MATRIX_COLS 6
 
-/*
- * Keyboard Matrix Assignments
- *
- * Change this to how you wired your keyboard
- * COLS: AVR pins used for columns, left to right
- * ROWS: AVR pins used for rows, top to bottom
- * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
- *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
- *
- */
 #define MATRIX_ROW_PINS { C6, D7, E6, B4, B5, B6, B7 }
 #define MATRIX_COL_PINS { F7, F6, F5, F4, F1, F0 }
 
 /* COL2ROW, ROW2COL */
 #define DIODE_DIRECTION COL2ROW
-#define SELECT_SOFT_SERIAL_SPEED 5
```

Comment:
What makes you think this is useless? I'd be curious why they selected such a low speed.

Reply:
My best guess is a misunderstanding that 5 is the fastest speed. Nonetheless, can be reverted if that's preferred.

Reply:
Please, it has the potential to break their setup. (even though it likely wouldn't)